Code/IO/src/sitkImageReaderBase.cxx

```diff
@@ -214,7 +214,7 @@ ::GetPixelIDFromImageIO( const ImageIOBase *iobase,
   if (numberOfComponents == 1 &&
       ( pixelType == itk::ImageIOBase::SCALAR || pixelType == itk::ImageIOBase::COMPLEX ) )
     {
-    outPixelType = this->ExecuteInternalReadScalar( componentType );
+    outPixelType = this->ExecuteInternalReadScalar( static_cast<int>(componentType) );
```

Comment:
@blowekamp I'm curious why the function signature is not changed so that component type is matched to the intent of "CommonEnums::IOComponent".  

Perhaps it is for backward compatibility?  If so I'd recommend 
```
#if ITK < 5.1
using SimpleITKInternalComponentType = int;
#else
using SimpleITKInternalComponentType = CommonEnums::IOComponent;
#endif
```

Just a thought to ease migration in the future.

Reply:
In all of the SimpleITK public headers, there is no inclusion of an ITK header.

There are some forward declarations of types used as pointers. But for this case there the enums have sizes, I don't expect a forward declaration to work.